scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/ufqfpn.yaml

```diff
@@ -0,0 +1,58 @@
+UFQFPN-32-1EP_5x5mm_P0.5mm_EP3.5x3.5mm:
+  device_type: 'UFQFPN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://www.st.com/resource/en/datasheet/stm32g071k8.pdf'
+  ipc_class: 'ufqfpn' # 'qfn_pull_back'
```

Comment:
That is wrong. It is either `qfn_pull_back` or not set all all.
Please remove that line.